src/servers/database.ts

```diff
@@ -863,7 +863,9 @@ export class DatabaseServer extends SkeletonServer {
     blockID: number | Uint8Array,
     fullObject: boolean
   ): Promise<unknown[] | string[]> {
-    const idColumn = typeof blockID === 'number' ? 'id' : 'hash';
+    const idColumn = ['bigint', 'number'].includes(typeof blockID)
```

Comment:
also need to change output type for `_fetchCurrentBlockID`, `Promise<bigint>` 

https://github.com/aurora-is-near/aurora-relayer/blob/d7b2de6e4f0f879a6d4f54218c41dce500b0c4e4/src/servers/database.ts#L825